### ISSUE TITLE
Suggestion: Nav bar should be accessible during scrolling

### ISSUE THREAD

**Issue by goofballLogic** [markeh/NewsReader#1]
On a mobile device, having scrolled through the news results, you have to scroll all the way to the top of the page again if you want to change the search string. Can we make the nav bar remain always visible at the top of the page so you don't have to scroll up again?

**Comment by markeh** [comment]
Yes indeed, this can be done with the application of the navbar-fixed-top class to the navbar. Do you believe this is only an issue on mobile devices only or is a fixed nav bar desirable all / other devices?

**Comment by goofballLogic** [comment]
@markeh dunno - i only noticed it on mobile, but I guess that's because the paging is set quite low, so it's not noticeable on desktop

**Comment by goofballLogic** [comment]
Is this up on markc.xyz yet?

**Comment by markeh** [comment]
> Is this up on markc.xyz yet?

Not yet, will add it tonight
